cli/main.go

```diff
@@ -185,7 +228,54 @@ func main() {
 						owner = &ownerProp
 					}
 
-					execute(ctx, eigenpodAddress, beacon, node, "validator", out, owner, false)
+					eth, beaconClient, chainId := getClients(ctx, node, beacon)
+					validatorProofs, validatorIndices := GenerateValidatorProof(ctx, eigenpodAddress, eth, chainId, beaconClient)
+					if validatorProofs == nil || validatorIndices == nil {
+						return nil
+					}
+
+					jsonString, err := json.Marshal(validatorProofs)
+					PanicOnError("failed to generate JSON proof data.", err)
+
+					WriteOutputToFileOrStdout(jsonString, out)
```

Comment:
we should probably put the indices here too?

Reply:
yeah, very good point

we should write out both

will do in a separate PR, since this PR is just moving code